resources/assets/lib/beatmap-discussions/new-discussion.tsx

```diff
@@ -376,26 +375,23 @@ export class NewDiscussion extends React.Component<Props> {
     if (this.nearbyDiscussions.length === 0) return;
     const currentTimestamp = this.timestamp != null ? formatTimestamp(this.timestamp) : '';
     const timestamps = this.nearbyDiscussions.map((discussion) => (
-      linkHtml(
-        BeatmapDiscussionHelper.url({ discussion }),
-        formatTimestamp(discussion.timestamp) ?? '',
-        { classNames: ['js-beatmap-discussion--jump'] },
-      )
+      <a
+        key={discussion.id}
```

Comment:
keys not needed anymore?

Reply:
`key` is still needed to pass linting (also probably shouldn't be assuming another component is splitting up arrays to not need keys)

Reply:
I wonder if it's better to pass the key separately instead (passing as map or array) 🤔 that way the join function can use it and allow the react render optimization thing